service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/BaseDevice.java

```diff
@@ -345,6 +345,8 @@ public long getCloudToDeviceMessageCount()
      * @param forceUpdate - Boolean controlling if the update should be forced or not
      * @throws IllegalArgumentException if the provided argument is null
      */
+    // Supressing this warning since this method is used by public APIs in Registry Manager
+    @SuppressWarnings("DeprecatedIsStillUsed")
```

Comment:
Are those APIs also deprecated?

Reply:
I guess what I am asking is why is the property deprecated if the methods that use it are not ! (if that is the case)

Reply:
method itself is deprecated here.

Reply:
Does that answer your question?

Reply:
Talked offline.